pilot/pkg/model/service.go

```diff
@@ -831,6 +832,12 @@ func (ep *IstioEndpoint) DeepCopy() *IstioEndpoint {
 	return copyInternal(ep).(*IstioEndpoint)
 }
 
+var gatewayNames = sets.New("ingressgateway", "egressgateway")
```

Comment:
Do we have these as constants anywhere? Is this even truly a "standard"?

Reply:
Seems not a standard naming